lib/std/math/complex.zig

```diff
@@ -115,6 +115,11 @@ pub fn Complex(comptime T: type) type {
         pub fn magnitude(self: Self) T {
             return @sqrt(self.re * self.re + self.im * self.im);
         }
+
+        /// Returns the squared magnitude of a complex number.
+        pub fn sqrmag(self: Self) T {
```

Comment:
```suggestion
        pub fn squaredMagnitude(self: Self) T {
```

this name makes the doc comment redundant

Reply:
ok